doc/rest-api.md

```diff
@@ -607,6 +607,15 @@ Input (using a remote container, in push mode sent over the migration websocket
                    "container_only": true}                                              # Whether to migrate only the container without snapshots. Can be "true" or "false".
     }
 
+Input (using a backup):
```

Comment:
This seems a bit hackish, I'd find it cleaner if we made it like images, where if the content-type is a binary blob, we just read it. That way it's trivial to upload a backup through curl for example.

Reply:
Looks like the code was changed but the doc wasn't?

Reply:
fixed